unattended.sh

```diff
@@ -26,18 +26,22 @@ wizard_autoplay=${ENV_WIZARD_AUTOPLAY:-"false"} #Default the install wizard is n
 central_ip=${ENV_CENTRAL_IP:-$default_ip}       #Default central ip is the first of hostname -I
 
 function genpasswd() {
-	local _pwd
-	_pwd=$(date +%s%N | sha256sum | base64 | head -c 32)
+        local _pwd
 
-	echo "Random password generated for user [$1] is [$_pwd]" >>$tmp_passwords_file
+        PWD_STRING_LENGTH=16
+        PWD_SPECIAL_CHARS='a-zA-Z0-9\!\@\$\*\?'
```

Comment:
we should always keep the same indentation
```suggestion
  PWD_STRING_LENGTH=16
  PWD_SPECIAL_CHARS='a-zA-Z0-9\!\@\$\*\?'
```